utils/fixed_decimal/src/decimal.rs

```diff
@@ -1012,7 +1012,7 @@ impl FixedDecimal {
 
     /// Zero-pad the number on the left to a particular position.
     ///
-    /// Negative numbers have no effect.
+    /// Negative position numbers have no effect.
```

Comment:
```suggestion
/// The position of the negative sign for negative numbers has no effect.
```

Reply:
I think this won't render the way you are thinking. `# NOTE` creates a heading, and the indentation creates a monospace code block. You could consider

```rust
/// NOTE: The position of the negative sign for negative numbers has no effect.
```

but I think what Kartavya proposed first is fine.

Reply:
Sure,

Also, I think it is better to have something like this:
```rust
/// NOTE: Negative numbers as input has no effect.
```